src/test/java/com/decide/DecideTest.java

```diff
@@ -383,33 +383,50 @@ public void LIC8TestNotInCircle(){
 
 
     @Test
-    /**Tests false boundary values for LIC8 where the number of points
-     * can not be less than 5 and a_pts and b_pts can not be lower than 1.
-    */
+    /**
+     * Requirements: See `LIC9` documentation
+     * Contract:
+     *      Precondition: numPoints can not be less than 5: numPoints = 4,
```

Comment:
The preconditions should probably have the format `x is less than y` instead of just describing the boundary.

Reply:
Yes, should be fixed now!